src/library_fs.js

```diff
@@ -628,6 +628,19 @@ mergeInto(LibraryManager.library, {
       mode |= {{{ cDefine('S_IFDIR') }}};
       return FS.mknod(path, mode, 0);
     },
+    // Creates a whole directory tree chain if it doesn't yet exist
+    mkdir_tree: function(path, mode) {
```

Comment:
for consistency with other function names, how about `mkdirTree`? Or `makeTree`?